K8s-deployment/Charts/auditing-server/Jenkinsfile

```
@@ -0,0 +1,23 @@
+pipeline {
+        agent {
+    node {
+      label 'slave1'
+    }
+  }
+  stages {
+      stage('Kubescape Report for Auditing Server'){
+          steps {
+              script{
+               sh 'helm template -f K8s-deployment/Charts/auditing-server/values.yaml -f K8s-deployment/Charts/auditing-server/example-azure-resource-values.yaml K8s-deployment/Charts/auditing-server  > auditing-server.yaml'
+               sh 'kubescape scan -t 25 -vv auditing-server.yaml --format pretty-printer --output auditing-report'
+              }
+          }
+          post {
+                always {
+               publishHTML([allowMissing: true, alwaysLinkToLastBuild:  true, keepAll: true, reportDir: '/var/lib/jenkins/workspace/kubescape-auditing/', reportFiles: 'auditing-report.txt', reportName: 'Kubescape Scan Report for Auditing Server'])
```

I don't think we need to allow missing reports anymore since we've separated all the pipeline executions. Each build must have a report associated with it whether it passes or fails. 
```suggestion
               publishHTML([allowMissing: false, alwaysLinkToLastBuild:  true, keepAll: true, reportDir: '/var/lib/jenkins/workspace/kubescape-auditing/', reportFiles: 'auditing-report.txt', reportName: 'Kubescape Scan Report for Auditing Server'])
```